docs/stories/box.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import {storiesOf} from '@storybook/react'
+import {Box, Info} from '@ambler/andive'
+
+import Showcase from './showcase'
+
+const shortLabel = 'Fan de Phil Collins'
+
+storiesOf('API|Box', module).add('Default', () => {
+  return (
+    <Showcase>
+      <Box style={{background: 'white'}}>
+        <Info.Label label={shortLabel} />
```

Comment:
I'd prefer a Typography.Body1 because we'll deprecate the Info.Label in coming weeks.